src/ansys/fluent/core/session.py

```diff
@@ -25,11 +25,14 @@
 def parse_server_info_file(filename: str):
     with open(filename, encoding="utf-8") as f:
         lines = f.readlines()
-    ip_and_port = lines[0].strip().split(":")
-    ip = ip_and_port[0]
-    port = int(ip_and_port[1])
-    password = lines[1].strip()
-    return ip, port, password
+    try:
+        ip_and_port = lines[0].strip().split(":")
+        ip = ip_and_port[0]
+        port = int(ip_and_port[1])
+        password = lines[1].strip()
+        return ip, port, password
+    except Exception:
```

Comment:
@prmukherj the correct catch-all type is `BaseException`.

Reply:
Note that, catching `BaseExecption` will shallow Ctrl+C. Catching `Exception` is generally fine for all coding logic level exception.